opcua_dbus.c

```diff
@@ -192,9 +233,182 @@ bool dbus_temp_unpack_signal(GVariant *parameters, uint32_t *subscription_id, do
     return true;
 }
 
-void dbus_connect_g_signal(GCallback func)
+void dbus_connect_temp_g_signal(GCallback func)
 {
-    assert(NULL != dbusproxy);
+    assert(NULL != dbusproxy_temp);
     assert(NULL != func);
-    g_signal_connect(dbusproxy, "g-signal", func, NULL);
+    g_signal_connect(dbusproxy_temp, "g-signal", func, NULL);
+}
+
+void dbus_connect_ports_g_signal(GCallback func)
+{
+    assert(NULL != dbusproxy_ports);
+    assert(NULL != func);
+    g_signal_connect(dbusproxy_ports, "g-signal", func, NULL);
+}
+
+/**
+ * get_number_of_ioports:
+ * @brief Get number of I/O ports from dbus.
+ * @return TRUE if successful.
```

Comment:
I like documentation and comments, but we should have these on all functions if we have it here. I'd suggest removing it for this PR and in that case have a separate PR that handles all that documentation. But I wouldn't oppose to keeping this without comments since the name of the function is good and clearly states what it is doing and what it delivers.